package.json

```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "class-variance-authority": "^0.7.0",
     "clsx": "^2.1.1",
+    "gsap": "^3.12.5",
```

Comment:
I think it will be better to first create app without any animations and then add them